Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -1044,11 +1037,9 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: 'Logs off the user by using the logoff command for the active user
-        session''s ID.
+      description: 'Logs off the user by using the logoff command for the active user session''s ID.
 
-        Note: the regex relies on the fact that interactively logged in users will
-        have an active "console" session in Windows machines.'
+        Note: the regex relies on the fact that interactively logged in users will have an active "console" session in Windows machines.'
```

Comment:
Note: The regex depends on the assumption that users logged in interactively will have an active "console" session on Windows machines.